docs/OMS-PostgreSQL-Instructions.md

```diff
@@ -0,0 +1,32 @@
+# PostgreSQL Log Monitoring Solution for Operations Management Suite
+
+1. Setup a Linux(Ubuntu/Redhat) machine and [install PostgreSQL](https://www.postgresql.org/download/linux/).
+
+2. Download and Install [OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux) on the machine. 
+
+3. Configure PostgreSQL to [generate logs](https://www.postgresql.org/docs/current/static/runtime-config-logging.html).
+
+4. Verify and update the PostgreSQL log file path in the configuration file `/etc/opt/microsoft/omsagent/conf/omsagent.d/postgresql_logs.conf`
+
+  ```
+  <source>
+    ...
+    path <PostgreSQL-file-path-for-logs>
+    ...
+  </source>
+  ```
+  If postgresql_logs.conf is not present in the above location, move it:
```

Comment:
Have you tested this on other supported machines besides Ubuntu/Redhat? If so, you show say "any supported Linux machine" instead of being more specific. If not, you should test on other supported distros.